scripts/mkcloud

```diff
@@ -1071,6 +1071,9 @@ Optional
         Enable debug level logging in barclamp proposals for all proposals.
     want_magnum=1 (default=0)
         It's not deployed by default. Set to 1 to deploy magnum.
+    magnum_trusts (default=0)
```

Comment:
I'd prefer want_magnum_trusts to have the common want_ scheme of things. also this means that the variable is actually exported to qa_crowbarsetup.sh which I think now it isn't.